osg-word-count/Dockerfile

```diff
@@ -13,8 +13,9 @@ RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add -
     && apt-get update \
     && apt-get install -y irods-icommands
 
-# Install the wrapper script.
+# Install the wrapper script and the script to upload the output files.
 ADD wrapper /usr/bin/wrapper
```

Comment:
(I am leaving a second review and comment to test the github-slack integration for this repo)